substrate/frame/revive/src/exec.rs

```diff
@@ -753,6 +754,80 @@ impl<T: Config> CachedContract<T> {
 	}
 }
 
+/// Function to transfer balance including the dust between accounts.
+fn transfer_with_dust<T: Config>(
+	from: &AccountIdOf<T>,
+	to: &AccountIdOf<T>,
+	value: BalanceWithDust<BalanceOf<T>>,
+) -> DispatchResult {
```

Comment:
Why? It is still only used in a single function? Why clutter the global scope? Why cause code churn? Or am I missing something?